### DISCUSSION TITLE
GitHub codespace keeps crashing

### DISCUSSION THREAD

**Discussion by VenomousSteam81** [community/community#40635] [Codespaces]
### Select Topic Area

Bug

### Body

My GitHub Codespace keeps crashing and I don't know why. I'm developing a Discord bot in it, and it almost always crashes when I try to run it, and the only way to fix it is by shutting it off from the GitHub Codespaces page. Can anyone please help?

**Comment by ossanna16** [comment]
Hi there @VenomousSteam81 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by LisMarEsc89** [comment]
**Apologies FOR ALL AND ANY INCONVENIENCES 
I''M ONLY A MONTH INTO THIS GITS¬HUB BUT APPARENTLY VIP BECAUSE ALL THESE LINKS THAT  LINK TO MY ACCOUNTS OR THERE IS ALREADY A PROFILE ON ME IN IT
THAT I NOT EVEN CONCIOUS TO EVEN REMEMBERING AS TO EVER  CREATING THEM **

CODE SPACE IS NOT SUPPOSE TO BE PUBLIC YET.NOT EVEN SURE IF AT ALL.
Will keep you updated or the team , if they can. 
Thank you
